packages/inquirer/package.json

```diff
@@ -60,15 +60,19 @@
   "license": "MIT",
   "dependencies": {
     "@inquirer/figures": "^1.0.3",
-    "@ljharb/through": "^2.3.13",
     "ansi-escapes": "^4.3.2",
-    "chalk": "^5.3.0",
     "cli-cursor": "^3.1.0",
     "cli-width": "^4.1.0",
     "external-editor": "^3.1.0",
-    "lodash": "^4.17.21",
+    "lodash.clone": "^4.5.0",
```

Comment:
do all those separate lodash dependencies in total still smaller than entire lodash? I'd assume there is at least some overlap between them.

probably some of them, such as set/get could be replaced by something like dot-prop

Reply:
pkg-size seems to suggest this:

- lodash 1.4MB
- this group of packages 199KB

Reply:
TBH, this one I've got PR wanting to move, then PR to move back to lodash, etc (feel free to search the issues history.) Lodash can be cached and often merged with other dependencies - so we're not comparing the same things when comparing size in isolation.